mix.exs

```diff
@@ -29,6 +42,19 @@ defmodule MongoosePush.Mixfile do
   #
   # Type "mix help deps" for more examples and options
   defp deps do
-    []
+    [
+     {:pigeon, git: "https://github.com/rslota/pigeon.git", tag: "6d1e4e3"},
```

Comment:
I see `rslota/pigeon` is a fork from `cstar/pigeon`. Are you @rslota going to send a PR to upstream?